.github/workflows/issue-milestone-collector.yml

```diff
@@ -208,11 +208,11 @@ jobs:
       run: |
             mkdir -p -v $SCRIPT_DIR/$STREAM_DIR $SCRIPT_DIR/$MAPPING_DIR $SCRIPT_DIR/$DATA_DIR
 
-    - name: Download Stream Artifacts
-      uses: actions/download-artifact@v3
-      with:
-        name: ${{ env.STREAM_NAME }}
-        path: ${{ env.SCRIPT_DIR }}/${{ env.STREAM_DIR }}
+    # - name: Download Stream Artifacts
+    #   uses: actions/download-artifact@v3
+    #   with:
+    #     name: ${{ env.STREAM_NAME }}
+    #     path: ${{ env.SCRIPT_DIR }}/${{ env.STREAM_DIR }}
```

Comment:
Commenting out these lines will prevent these files from being commited. These files are causing the JSON Check failures. 

Also, the 2 json files within `scripts/issue-milestone-collector/github-artifacts/api-payload/` need to be deleted. The `api-payload` directory can be deleted